lib/ansible/module_utils/urls.py

```diff
@@ -1369,6 +1371,7 @@ def open(self, method, url, data=None, headers=None, use_proxy=None,
         :kwarg ca_path: (optional) String of file system path to CA cert bundle to use
         :kwarg unredirected_headers: (optional) A list of headers to not attach on a redirected request
         :kwarg decompress: (optional) Whether to attempt to decompress gzip content-encoded responses
+        :kwarg use_netrc: (optional) Boolean determining whether to use credentials from ~/$USER/.netrc file
```

Comment:
```suggestion
        :kwarg use_netrc: (optional) Boolean determining whether to use credentials from ~/.netrc file
```
(the first suggestion wasn't actually correct, this one is)

Reply:
ok, got it.